rtgui/adjuster.h

```diff
@@ -75,28 +73,25 @@ class Adjuster : public Gtk::Grid
     double logPivot;
     bool logAnchorMiddle;
 
-    double shapeValue (double a);
-    void   refreshLabelStyle ();
+    double shapeValue (double a) const;
     double2double_fun value2slider, slider2value;
 
-    double getSliderValue();
+    double getSliderValue() const;
     void setSliderValue(double val);
 
 public:
 
     int delay;
 
-    Adjuster (Glib::ustring vlabel, double vmin, double vmax, double vstep, double vdefault, Gtk::Image *imgIcon1 = nullptr, Gtk::Image *imgIcon2 = nullptr, double2double_fun slider2value = nullptr, double2double_fun value2slider = nullptr);
+    Adjuster (Glib::ustring vlabel, double vmin, double vmax, double vstep, double vdefault, Gtk::Image *imgIcon1 = nullptr, Gtk::Image *imgIcon2 = nullptr, double2double_fun slider2value_ = nullptr, double2double_fun value2slider_ = nullptr);
     ~Adjuster () override;
 
     // Add an "Automatic" checkbox next to the reset button.
     void addAutoButton(Glib::ustring tooltip = "");
-    // Remove the "Automatic" checkbox next to the reset button.
-    void delAutoButton();
     // Send back the value of og the Auto checkbox
     bool getAutoValue ()
     {
-        return automatic != nullptr ? automatic->get_active () : false;
+        return automatic ? automatic->get_active() : false;
```

Comment:
Oh well, let's move implementations to the implementation file.

Reply:
Done